fuel/schemes.py

```diff
@@ -1,10 +1,9 @@
 from abc import ABCMeta, abstractmethod
 
 import numpy
-from picklable_itertools import chain, repeat, imap, _iter
+from picklable_itertools import chain, repeat, imap, xrange, iter_
```

Comment:
Two things: 

* If you iterate on it, you needn't call `iter_()`; you'll get a `range_iterator` automatically.
* Some Python 3 features (`.count()`, `.index()`) are available in this implementation that aren't generally.

#29 makes adding it here irrelevant, since `partition_all`, etc. call `iter_` internally, so I can remove it if you like. It does lead to terser code in places where you'd need an additional `iter_` call though.